packages/core/src/libraries/user.ts

```diff
@@ -1,5 +1,5 @@
-import type { User, CreateUser, Scope } from '@logto/schemas';
-import { Users, UsersPasswordEncryptionMethod } from '@logto/schemas';
+import type { User, CreateUser, Scope, BindMfa, MfaVerification } from '@logto/schemas';
```

Comment:
didn't see any comment in mfa-related types. better to add **TSDoc** comments to them

Reply:
created LOG-7324 to track